backend/app/controllers/spree/admin/variants_controller.rb

```diff
@@ -31,12 +31,10 @@ def new_before
       end
 
       def collection
-        @deleted = (params.key?(:deleted) && params[:deleted] == "on") ? "checked" : ""
```

Comment:
I don't think so. It was only used for filtering, and that's been entirely revamped (we already "lost" `@option_types` and `@option_values`).